packages/gitbook/src/components/Header/HeaderLogo.tsx

```diff
@@ -21,6 +21,9 @@ export async function HeaderLogo(props: HeaderLogoProps) {
     return (
         <Link
             href={linker.toAbsoluteURL(linker.toPathInSite(''))}
+            // This link should always open in the same tab
+            // so we set the target to _self.
+            target="_self"
```

Comment:
```suggestion
href={linker.toPathInSite('')}
```

Reply:
I don't think we should hack it with `target="_self"` here; we should have a proper relative link and be fine with it